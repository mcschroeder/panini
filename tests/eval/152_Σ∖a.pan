import axioms

g1 : {s:char| s = 'a'}

f152 : {s:string|?} -> unit
f152 = \s:string.
  let t = g1 in
  let c = charAt s 0 in
  let p = eqChar c t in
  let p2 = not p in
  let _ = assert p2 in
  let n = length s in
  let p3 = eq n 1 in
  assert p3
