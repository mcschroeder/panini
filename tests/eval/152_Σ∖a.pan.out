f152 : {s:𝕊 | s ∈ [^a]} → 𝟙